test/.eslintrc.json

```diff
@@ -1,5 +1,8 @@
 {
   "rules": {
-    "no-unused-expressions": "off"
+    "no-unused-expressions": "off",
+    "max-len": "off",
+    "no-unused-vars": "off",
+    "no-shadow": "off"
```

Comment:
Im not done amending my commit. Fixing the next thing you mentioned will get rid of the need for this.